lib/tasks/after_tasks.rake

```diff
@@ -582,6 +582,23 @@ namespace :After do
     end
   end
 
+  desc "Enforce HTTPS where available for embedded media from ning.com and vidders.net"
+  task(enforce_https: :environment) do
+    Chapter.find_each do |chapter|
+      if chapter.id % 1000 == 0
```

Comment:
Should I do anything about the style violations in this file? Since I just copy-pasted the similar task and modified it. And if so, should I also change the other task?

Reply:
There's a lot of code around with style violations, since we weren't always consistent in using style checkers. In general, we fix violations on lines added/modified in the current pull request, and leave the existing ones alone.

If you'd like to check for violations locally, you can [run `rubocop`](https://github.com/otwcode/otwarchive/wiki/Coding-standards#Previewing_Hound_comments). It's not a perfect solution since it reports everything in a file, not just the changed lines, but it can save you from the back-and-forth with the Hound bot.